Formula/atomist-cli.rb

```diff
@@ -3,14 +3,8 @@
 class AtomistCli < Formula
   desc "The Atomist CLI"
   homepage "https://github.com/atomist/cli#readme"
-  url "https://registry.npmjs.org/@atomist/cli/-/@atomist/cli-1.6.0.tgz"
-  sha256 "0c00e070c8525df747676ea30241c772631c622664b26146f313fe1019778adb"
-
-  bottle do
```

Comment:
Please don't remove the `bottle` block, our merging process will update it automatically.

Reply:
Doesn't it also automatically add it if it is not there?

Reply:
It does, most of the time. But let's keep the changes to the minimal, so please restore it. (and squash into a single commit)

Reply:
Okay. We just figured having no information was better than wrong information.